### DISCUSSION TITLE
halo工程加载后怎么是这样的，没有办法本地运行

### DISCUSSION THREAD

**Discussion by zuoer96** [halo-dev/halo#4545] [Q&A]
![image](https://github.com/halo-dev/halo/assets/31268184/f0040906-9863-420f-9bbd-46a7e70884ce)

**Comment by JohnNiang** [comment]
我这边没有任何问题。建议刷新 Gradle 依赖并构建项目。

<img width="1327" alt="image" src="https://github.com/halo-dev/halo/assets/16865714/a1ca9f19-14ba-4ac7-aeac-ae55f9044e54">
